src/python/pants/backend/helm/util_rules/post_renderer.py

```diff
@@ -45,10 +48,29 @@ def debug_hint(self) -> str | None:
         return self.field_set.address.spec
 
 
+@rule_helper
+async def _obtain_custom_image_tags(
+    address: Address, union_membership: UnionMembership
+) -> DockerImageTags:
+    wrapped_target = await Get(
+        WrappedTarget, WrappedTargetRequest(address, description_of_origin="<infalible>")
```

Comment:
This is not important enough to be worth another CI round, but if you're making other changes then the customary spelling is "infallible" (and this might matter a tiny bit if grepping for this string in the future).